plotting/plotter_3d.py

```
@@ -0,0 +1,375 @@
+            else:
+                self.basemap = basemap.load_map(
+                    'lcc', self.centroid, height, width)
+        elif abs(self.centroid[1] - self.bounds[1]) > 90:
+
+            height_bounds = [self.bounds[0], self.bounds[2]]
+            width_bounds = [self.bounds[1], self.bounds[3]]
+            height_buffer = (abs(height_bounds[1]-height_bounds[0]))*0.1
+            width_buffer = (abs(width_bounds[0]-width_bounds[1]))*0.1
+
+            if abs(width_bounds[1] - width_bounds[0]) > 360:
+                raise ClientError(gettext("You have requested an area that exceeds the width of the world. \
```

Ruff (0.12.2)

Undefined name `ClientError`

(F821)